README.md

```diff
@@ -39,11 +39,14 @@ For more details on how it works, read the [documentation](https://docs.nativesc
 ## Build Prerequisites
 Following are the minimal prerequisites to build the runtime package.
 
-* Install the latest [Android Studio ](https://developer.android.com/studio/index.html).
-* Install Android API Level 22, 23, 24, 25 from Android Studio -> Tools -> Android -> SDK Manager
-* Android NDK 15c - [NDK Archives](https://developer.android.com/ndk/downloads/older_releases.html) or [NDK Downloads](https://developer.android.com/ndk/downloads/index.html) if you can't find it in the archives.
-* Download Android Support Repository through the Android SDK Manager
-* Download Build Tools 25.0.2 through the Android SDK Manager.
+* Install the latest [Android Studio](https://developer.android.com/studio/index.html).
+* From the SDK Manager (Android Studio -> Tools -> Android -> SDK Manager) install the following components:
+    * Android API Level 22, 23, 24, 25
```

Comment:
https://github.com/NativeScript/android-runtime/blob/74a1cde822bba9340a21feb99f0ebbb9912502c3/test-app/app/build.gradle#L63 compileSdk version defaults to 26, maybe we should recommend the installation of 23, 24, 25, 26, others could be recommended, but optional.